assemblies/assembly_decision-manager/main.adoc

```diff
@@ -10,11 +10,10 @@ include::_artifacts/document-attributes.adoc[]
 include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
-As a business analyst or rules developer, you can use {CENTRAL} in {PRODUCT} to design a variety of decision services. In this tutorial, you will create and test a driver's license suspension scenario.
```

Comment:
Items not in PR:
* Search all instances of "Assets view" and make sure all are bold, "**Assets** view".
* Search "NOTE: The violations" and make that note a line break down in the cell and use unicode for "The `violations` field ... to `List` to ..."
* Search all "Guided Rule wizard" and change to "guided rules designer". Also all instances of capitalized "Guided Rule" and likely need to change to "guided rule" or "guided rules", unless referring directly to how it appears in the UI (like the "Create a new Guided Rule window"). Same with "Guided Decision Tables designer," which should be lowercase. Your judgement, but point is that the terminology standard for designers is "guided rules designer" (plural), "guided decision tables designer", etc. No such thing as a Guided Rule wizard in central.
* Search all instances of "enter:" and change to "enter the following values:" or whatever makes sense to complete it. That's just what you use elsewhere, so good to be consistent.
* In "Creating a traffic violation guided decision table": Change to "Click *Ok* to open the *Create new Guided Decision Table* window". Same with the screenshot label. What that shows isn't the guided decision tables designer, which is the entire designer palette, like all other designers in central.
* Sometimes you use unicode in titles, like "Inserting `Fine Amount and Points` columns", and sometimes you don't, like "Inserting Violation Type columns". Pick one way or the other.
* In all three "Testing the ... scenario" sections (5.1-5.3): Change bottom note to "the **Reporting** section ... a **Success** message."

Reply:
Mostly done